src/Tizen.NUI/src/public/Common/PropertyNotification.cs

```diff
@@ -66,24 +66,29 @@ internal PropertyNotification(global::System.IntPtr cPtr, bool cMemoryOwn) : bas
         {
             add
             {
-                // Restricted to only one listener
                 if (propertyNotificationNotifyEventHandler == null)
                 {
-                    propertyNotificationNotifyEventHandler += value;
-
-                    propertyNotificationNotifyEventCallbackDelegate = new NotifyEventCallbackDelegate(OnPropertyNotificationNotify);
-                    this.NotifySignal().Connect(propertyNotificationNotifyEventCallbackDelegate);
+                    propertyNotificationNotifyEventCallback = OnPropertyNotificationNotify;
+                    using PropertyNotifySignal signal = new PropertyNotifySignal(Interop.PropertyNotification.NotifySignal(SwigCPtr), false);
```

Comment:
Great, It's very reasonable to use `using` statement here not to make warning message :)